src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -67,7 +71,7 @@
     [path]
     (mapcat
       #(fs/glob (fs/file path %))
-      ["*.ini" "*.conf" "*.json" "*.properties" "*.edn"])))
+      ["*.ini" "*.conf" "*.json" "*.properties" "*.edn" ".yaml" ".yml"])))
```

Comment:
I think this needs to be `"*.yaml" "*.yml"`, since `fs/glob` matches files using globbing.

I don't think there's currently anything testing this code path (parsing yaml files in a directory when that directory is passed as the config). It would be good to add a yaml file to `dev-resources/config/mixeddir` or one of the other directories that gets parsed to ensure that this works.